src/PrestaShopBundle/Resources/config/services/adapter/product/_seo.yml

```diff
@@ -5,6 +5,6 @@ services:
   prestashop.adapter.product.options.redirect_target_provider:
     class: PrestaShop\PrestaShop\Adapter\Product\Options\RedirectTargetProvider
     arguments:
-      - '@prestashop.adapter.product.repository.product_preview_repository'
+      - "@prestashop.adapter.product.image.repository.product_image_repository"
```

Comment:
Why switching to double quotes here ?

Reply:
dont't know why ! ;-) bad copy -paste ??